firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -23,6 +23,7 @@ void setupConsoleFunctions() {
     // particle console function declarations, belongs in setup() as per docs
     Particle.function("Force_Reset", force_reset);
     Particle.function("Turn_Debugging_Publishes_On_Off", toggle_debugging_publishes);
+    Particle.function("Change_Occupant_Detection_Window", window_size_set);
```

Comment:
I like this name! Maybe use `occupant_detection_timer` or something like that for the variables.

Reply:
yep, that makes more sense, I'll change the other variable names